backend/src/main/kotlin/app/ehrenamtskarte/backend/common/webservice/GraphQLContext.kt

```diff
@@ -0,0 +1,10 @@
+package app.ehrenamtskarte.backend.common.webservice
+
+import app.ehrenamtskarte.backend.auth.webservice.JwtPayload
+import com.expediagroup.graphql.execution.GraphQLContext
+
+data class GraphQLContext(
+    val jwtPayload: JwtPayload?
+) : GraphQLContext {
+    val isSignedIn get() = jwtPayload != null
```

Comment:
should we verify the payload here? Or is the payload never in the context if it is not verified?

Reply:
> Or is the payload never in the context if it is not verified?

This